doc/sphinx/source/contributing/rules.md

```diff
@@ -1,26 +1,26 @@
 ```eval_rst
 .. _rules:
 ```
-# Code development 
+# Code development
 
-Code development is carried out using Github. 
+Code development is carried out using Github.
 For more information on the Git workflow that NNPDF adopts, see the [Git and GitHub](./git.md) section.
 
-## Rules 
+## Code contributions
 
-**Never commit modifications directly to the master version of the code! Instead, 
-open a new branch of the code and make modifications on that branch. To merge
-changes to master a pull request (PR) should be opened.**
+Code contributions should be presented in the form of [Pull
+Requests](https://github.com/NNPDF/nnpdf/pulls)(PRs) to the repository.
+Never commit modifications directly to the master version of the code. Instead,
+create a new branch and make modifications on it.
```

Comment:
Maybe we can disable it altogether @Zaharid @scarrazza? (so that only admins can do it)

Reply:
I think that was the case at some point, but github has a way to toggle stuff unexpectendly for you.

Reply:
Maybe it's because everyone is an admin? Because we can do: Include administrators Enforce all configured restrictions above for administrators.

Reply:
Shouldn't we have it such that to contribute you need to create a fork with the modification?

Reply:
Well, external people can only do PRs by creating branches.